src/fprime_gds/flask/static/js/vue-support/channel.js

```diff
@@ -81,29 +81,43 @@ Vue.component("channel-table", {
         /**
          * Use the row's values and bounds to colorize the row. This function will color red and yellow items using
          * the boot-strap "warning" and "danger" calls.
+         * @param item: channel item
          * @return {string}: style-class to use
          */
-        calculateRowStyle() {
+        calculateRowStyle(item) {
+            if (item.val == null) {
+                // No channel value yet
+                return "";
+            }
             let template = _dictionaries.channels[item.id];
-            let value = this.channel.value;
+            // Check if any bounds are actually set
+            if ((template.low_red == null) && (template.low_orange == null) && (template.low_yellow == null) &&
+                (template.high_red == null) && (template.high_orange == null) && (template.high_red == null)) {
```

Comment:
```suggestion
                (template.high_red == null) && (template.high_orange == null) && (template.high_yellow == null)) {
```

Duplicated `high_red`.